chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/listing/reviewer/edition2015/OptionalStandardReviewer.java

```diff
@@ -0,0 +1,48 @@
+package gov.healthit.chpl.validation.listing.reviewer.edition2015;
+
+import org.ff4j.FF4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import gov.healthit.chpl.FeatureList;
+import gov.healthit.chpl.domain.CertificationResult;
+import gov.healthit.chpl.domain.CertificationResultTestStandard;
+import gov.healthit.chpl.domain.CertifiedProductSearchDetails;
+import gov.healthit.chpl.util.ErrorMessageUtil;
+import gov.healthit.chpl.util.Util;
+import gov.healthit.chpl.validation.listing.reviewer.Reviewer;
+
+@Component("optionalStandardReviewer")
+public class OptionalStandardReviewer implements Reviewer {
+    private ErrorMessageUtil msgUtil;
+    private FF4j ff4j;
+
+    @Autowired
+    public OptionalStandardReviewer(ErrorMessageUtil msgUtil, FF4j ff4j) {
+        this.msgUtil = msgUtil;
+        this.ff4j = ff4j;
+    }
+
+    @Override
+    public void review(CertifiedProductSearchDetails listing) {
+        if (ff4j.check(FeatureList.OPTIONAL_STANDARDS)) {
+            listing.getCertificationResults().stream()
+            .filter(cert -> (cert.getTestStandards() != null && cert.getTestStandards().size() > 0))
```

Comment:
I think I'm confused about this reviewer... I know optional standards are replacing test standards and there is currently some overlap, and I saw this work in your demo. But this is checking against testStandards only as far as I can tell. Where is the code putting optional standards in the testStandards array? I can't find it but am probably missing it. Should there also be validation against certResult.getOptionalStandards()? What about checking that the optional standard chosen is one of the ones allowed for the criteria (yes someone would have to use the API for this).

Reply:
This one was poorly named. I renamed it as a separate reviewer that we might be able to delete later, and added a reviewer specifically for Optional Standards (including some tests for it)